### DISCUSSION TITLE
Function PathFinder

### DISCUSSION THREAD

**Discussion by adraconbytel** [orientechnologies/orientdb#10053] [Q&A]
In the sources I can find the function below, I would like to use it in SQL Query, but it seem to not be available .. why ?
graphdb/src/main/java/com/orientechnologies/orient/graph/sql/functions/OSQLFunctionPathFinder.java

**Comment by tglman** [comment]
Hi,

That is an base implementation that is then by a real implementation that is dijkstra so use that fonctionas described here:
https://orientdb.com/docs/3.2.x/sql/SQL-Functions.html#dijkstra
